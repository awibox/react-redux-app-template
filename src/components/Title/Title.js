import React from 'react';
import PropTypes from 'prop-types';
// Styles
import styles from './Title.scss';

const Title = (props) => (
  <div className={styles.title}>
    {props.children}
  </div>
);

Title.propTypes = {
  children: PropTypes.any,
};

export default Title;
