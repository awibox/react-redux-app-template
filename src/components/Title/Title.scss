@import '../../styles/variables';

.title {
  font-size: 22px;
  line-height: 26px;
  padding-bottom: $defaultIndent;
  text-align: left;
}
