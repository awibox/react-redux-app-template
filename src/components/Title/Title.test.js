import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title';

describe('Title', () => {
  it('should render correctly', () => {
    const component = shallow(<Title>Test title</Title>);
    expect(component).toMatchSnapshot();
  });
});
