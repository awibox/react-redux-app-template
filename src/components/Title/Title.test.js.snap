// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<div
  className="title"
>
  Test title
</div>
`;
